doc/nrf/app_dev/device_guides/nrf54l/vpr_flpr.rst

```diff
@@ -0,0 +1,29 @@
+.. _vpr_flpr_nrf54l:
+
+Working with the FLPR core
+##########################
+
+The nRF54L15 SoC has a dedicated RISC-V CPU (VPR), a fast lightweight peripheral processor (FLPR) designed for software-defined peripherals (SDP).
+This core currently supports the following peripherals:
+
+* GPIO
+* GPIOTE
+* GRTC
+* TWIM
+* UARTE
+* VPR
+
+Running the FLPR core
+*********************
+
+VPR is optimized for implementing SDP.
+The software-defined peripheral running on VPR does not start on its own, but must be started by the application core processor.
+
+.. To be outlined how
```

Comment:
```suggestion
Running the FLPR core
*********************

General usage
-------------
FLPR can be used as a generic core, running full Zephyr kernel. In this case, FLPR target can be built similarly to APP core. Note that APP core build needs to include an overlay enabling FLPR coprocessor, so the code that starts FLPR will be included.

Software Defined Peripherals
----------------------------
```
VPR is optimized for implementing Software Defined Peripherals (SDP). Full documentation on SDP can be found here: <TODOLINK>
```

Reply:
I added this but would this be enough for the user to know how to proceed? Shouldn't we provide some more details on it?

Reply:
I was thinking about the SDP documentation showing how to proceed in that topic.
When it comes to General usage, this is just using full Zephyr on FLPR, so not sure what we could add here.